class.jetpack.php

```diff
@@ -4032,6 +4032,20 @@ function admin_page_load() {
 					wp_safe_redirect( Jetpack::admin_url( array( 'page' => $redirect ) ) );
 				}
 				exit;
+			case 'onboard' :
+				if ( ! current_user_can( 'manage_options' ) ) {
+					wp_safe_redirect( Jetpack::admin_url( 'page=jetpack' ) );
+				} else {
+					$token = Jetpack::create_onboarding_token();
+					$url = $this->build_connect_url( true );
```

Comment:
What's the meaning of the `true` arg here? And why aren't we using `$token` anywhere? :thinking:

Reply:
> What's the meaning of the `true` arg here?

It allows us to obtain the raw, non-escaped connection URL.

> And why aren't we using `$token` anywhere? :thinking: 

Good catch - the var declaration was a remnant from the previous implementation. Removed in 5bbade9bde14d8f015d32ffde9513f2dd1fd2b18.

I'm keeping the `Jetpack::create_onboarding_token()` because it'll generate a token if one hasn't been generated yet.